flink-ai-flow/ai_flow/java/client/src/test/java/com/aiflow/client/AIFlowClientTest.java

```diff
@@ -0,0 +1,779 @@
+import com.aiflow.entity.ModelRelationMeta;
+import com.aiflow.entity.ModelVersion;
+import com.aiflow.entity.ModelVersionMeta;
+import com.aiflow.entity.ModelVersionRelationMeta;
+import com.aiflow.entity.ProjectMeta;
+import com.aiflow.entity.RegisteredModel;
+import com.aiflow.entity.WorkflowMeta;
+import com.aiflow.exception.AIFlowException;
+import com.aiflow.notification.client.EventWatcher;
+import com.aiflow.notification.entity.EventMeta;
+import com.aiflow.proto.Message;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+public class AIFlowClientTest {
+
+    private static final String LOCALHOST = "localhost";
+    private static final int SEC = 1000;
+
+    private static AIFlowClient client;
+    private static final Map EMPTY_MAP = Collections.emptyMap();
+
+    @BeforeAll
+    public static void beforeClass() throws Exception {
+        int port = Integer.valueOf(getProperties("port"));
+        client = new AIFlowClient(LOCALHOST + ":" + port, "default",
+                false, 1 * SEC, 1 * SEC, 1 * SEC);
+    }
+
+    @AfterAll
+    public static void afterClass() throws Exception {
+        client.stopListenEvent("default", "");
+    }
+
+
+
+    private static String getProperties(String key) throws IOException {
+        InputStream in = new Object() {
+            public InputStream getInputStream() {
+                return this.getClass().getClassLoader().getResourceAsStream("test.properties");
+            }
+        }.getInputStream();
+        Properties properties = new Properties();
+        properties.load(in);
+        return properties.getProperty(key);
+    }
+
+    public static void main(String[] args) throws IOException {
+        System.out.println(getProperties("port"));
+    }
+
+    // test dataset
+
+    @Test
+    public void testRegisterDataset() throws Exception {
+        String datasetName = "dataset_name";
+        client.registerDataset(datasetName, "csv", "good dataset", "mysql://",
+                new HashMap<String, String>() {{
+                    put("key1", "value1");
+                    put("key2", "value2");
+                }},
+                Arrays.asList("col1", "col2"),
+                Arrays.asList(DataType.STRING, DataType.STRING));
+        DatasetMeta notExistsDataset = client.getDatasetById(2L);
+        Assertions.assertNull(notExistsDataset);
+
+        DatasetMeta datasetById = client.getDatasetById(1L);
+        Assertions.assertEquals(datasetName, datasetById.getName());
+
+        DatasetMeta datasetByName = client.getDatasetByName(datasetName);
+        Assertions.assertEquals(1L, datasetByName.getUuid().longValue());
```

Comment:
import the `org.junit.jupiter.api.Assertions. assertEquals`, and this use `assertEquals`.